### PR TITLE
Add option to hide topbar

### PR DESCRIPTION
**Pull request by PhiloNL**
## Description
This PR adds new functionality that allows you to hide the entire topbar.

## Code style
- [ ?] `composer cs` command has been run. // This will modify a lot of files?

## Testing

- [x] Changes have been tested.

## Documentation

- [x] Documentation is up-to-date.

### CODE REVIEW

File: packages/panels/resources/views/components/layout/index.blade.php [hunk 1]
```diff
@@ -43,11 +43,14 @@ class="fi-sidebar-close-overlay fixed inset-0 z-30 bg-gray-950/50 transition dur
                 'flex' => filament()->hasTopNavigation() || (! filament()->hasNavigation()),
             ])
         >
-            {{ \Filament\Support\Facades\FilamentView::renderHook('panels::topbar.before') }}
 
-            <x-filament-panels::topbar :navigation="$navigation" />
+            @if(filament()->hasTopBar())
```
Comment (danharrin) [inline packages/panels/resources/views/components/layout/index.blade.php]:
```suggestion
            @if (filament()->hasTopBar())
```

**Review by danharrin — CHANGES_REQUESTED** [review]
Very sorry, but a tiny nitpick. The feature is called `topbar` and not `top bar` everywhere else, so the camel case should also be changed to be `topbar` instead of `topBar`. So it's consistent with `sidebar`.

**Comment by PhiloNL** [comment]
@danharrin no worries 😄 all set!

File: packages/panels/resources/views/components/layout/index.blade.php [hunk 2]
```diff
@@ -43,11 +43,14 @@ class="fi-sidebar-close-overlay fixed inset-0 z-30 bg-gray-950/50 transition dur
                 'flex' => filament()->hasTopNavigation() || (! filament()->hasNavigation()),
             ])
         >
-            {{ \Filament\Support\Facades\FilamentView::renderHook('panels::topbar.before') }}
 
-            <x-filament-panels::topbar :navigation="$navigation" />
+            @if (filament()->hasTopBar())
```
Comment (danharrin) [inline packages/panels/resources/views/components/layout/index.blade.php]:
```suggestion
            @if (filament()->hasTopbar())
```

File: packages/panels/docs/06-navigation.md [hunk 3]
```diff
@@ -328,6 +328,21 @@ public function panel(Panel $panel): Panel
 }
 ```
 
+### Disabling the top bar
```
Comment (danharrin) [inline packages/panels/docs/06-navigation.md]:
```suggestion
### Disabling the topbar
```

File: packages/panels/docs/06-navigation.md [hunk 4]
```diff
@@ -328,6 +328,21 @@ public function panel(Panel $panel): Panel
 }
 ```
 
+### Disabling the topbar
+
+You may disable top bar entirely by passing `false` to the `topbar()` method:
```
Comment (danharrin) [inline packages/panels/docs/06-navigation.md]:
```suggestion
You may disable topbar entirely by passing `false` to the `topbar()` method:
```

**Comment by danharrin** [comment]
Thanks Philo!

**Pull request merged**